pkgs/by-name/mo/motoc/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  openxr-loader,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "motoc";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "galister";
+    repo = "motoc";
+    rev = "refs/tags/v${version}";
```

```suggestion
    rev = "v${version}";
```

You don't need `refs/tags/`.

---

I prefer to be explicit, to the point of nitting on other PRs to add the `refs/tags/` prefix.